### ISSUE TITLE
YouTube & Twitch: My Revengence

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#15]
I like making video content. More specifically, my YouTube content on my adventures with gamedev, and my various attempts at livestreams. 

And just like how [I missed blogging](#) and fixed it by restarting my blog, I miss making videos and livestreams.

But why do I miss it? It's time consuming, it takes away from gamedev and other projects...what's the point? 

Let's unpack that by asking the [important questions](): _What value does making videos and livestreams being me_ and _what values does it bring you_? 

# What value does it bring me? 

For pre-recorded video content, I like the  challenge that comes with filming and editing a video. There's a kind of "visual thinking" that is required to make videos and I like how it forces me to practice that type of communication style. 

For livestreams, I really like keeping my live speaking skills up, and adding the extra video production challenges with it. Keeps me thinking on my toes and forcing myself to engage with my audience. 

In short, I like the challenge and appreciate the skills required to deliver the content. 

# What value does it bring you? 

This has been the important questions that I've struggled with the past while looking back through my videos. 

I think my YouTube content has some value by sharing my experiences, but they were missing something: my truth and my passion for video games. 

My truth is that I'm not trying to become a professional game developer, rather I'm trying to make a video game because I love the medium. When I'm not trying to make my own game, I'm learning about the industry, subculture, and history about video games. 

I mean I'm watching SGDQ 2021 in bed right now while I write this post because speed running is amazing and I find the speed running community truly inspiring. 

# Cool cool. So now what? 

Well, a couple of things. 

## Operation: Play My Collection

I'm going to start livestreaming, but not for code. I've wanted to try and play (not beat)each game in my personal collection to see what inspirarion lies within my own walls. 

Goal is to get a sense of each and and figure out _where's the fun_ in each game. 

I'm going to try it over the summer on my Twitch channel and see what happens. I have ideas on where else I want to take that, but for now I'm going to try and stream.

## My Non-Gamedev Story
As opposed to Twitch, YouTube is going to stay focused on game development, but it's going to get a little more honest. I hope that my new content will take some cues from my passion for the medium rather just my technical aspiraand progress. 

I'll elaborate when I have something ready, but for now know that I'm going try and make video content that shares my passion and not just my own story. 

# TL;DR; (Conclusion) 
All that to say, I miss making video content enough to get back into it with some changes to the format. To do that, I'm going to start with rebooting my Twitch channel with livestreams of me searching my own collection for hidden gems. 

YouTube will get new content too, but more on that when I have some pre-recorded content ready and waiting. 

If you're so inclined, subscribe to my YouTube and/or Twitch channel to get notified. 

If you prefer the blog, keep it up, as I'll keep posting here as things develop. 

Thanks for playing. 

~ DW

**Comment by davidwesst** [comment]
Closed with commit 69f18c8c2612979c5a7e2ccf0b79c9c83e66117a
